PopulateTools/gobierto
# Rescue AR Not found exception with a 404 page

Unexpected

## :v: What does this PR do?

This was a misconfiguration in the application. AR not found error was captured with a 404 error page, but a Rollbar was emitted. This PR avoids the Rollbar.

## :mag: How should this be manually tested?

Load a resource in draft:

- check the app returns the 404 page
- check no rollbar has been emitted
